jjb/device/device-sdk-c.yaml

```diff
@@ -1,20 +1,28 @@
 ---
+
 - project:
     name: device-sdk-c
-    project-name: device-sdk-c
+    project-name:  device-sdk-c
     project: device-sdk-c
-    build-node: centos7-builder-2c-1g
-    nexus-group-id: device-sdk-c
     mvn-settings: device-sdk-c-settings
-    archive-artifacts: '$WORKSPACE/device-sdk-c/build/release/*.gz'
-    pre_build_script: ''
-    git_url: '$GIT_URL/$PROJECT'
-    build_script: '$WORKSPACE/device-sdk-c/scripts/build.sh'
-    build_dir: '$WORKSPACE/device-sdk-c/scripts/build'
+    docker_name: docker-device-sdk-c
+    docker_root: ''
+    docker_build_args: '-f scripts/Dockerfile.alpine-3.7'
+    archive-artifacts: '**/release/*.tar.gz'
+    post_build_script: !include-raw-escape: shell/device_sdk_c_build.sh
```

Comment:
This should be added as a builder (probably to the appropriate base job), not as a post-build script.

Reply:
I dont know how to do that. Do you have examples of other jobs that do this?

Reply:
The base job in this case, `jjb/edgex-templates-docker.yaml`, is shared by a score of other jobs. This is a special case as the developer's have their own specialized build flow. I do not think it would be appropriate to add this one-off code the the base template. 🤷‍♂️

Reply:
Sorry, nevermind. This feature was confusingly named (post-build is a stage of the Jenkins job, but this is simply a build script that runs last during the build stage), but it does what you want it to do without violating how the Jenkins job should be formatted. This is fine.